context/AuthContextProvider.tsx

```diff
@@ -65,20 +72,22 @@ export const AuthContextProvider = ({
   };
 
   // get user
-  const getUser = cache(async () => {
+  const getUser = useCallback(async () => {
     if (!isSessionInLocalStorage()) {
       return;
     }
 
     try {
-      const userData = await account.get();
-      updateUser(userData.$id, userData.email);
+      const user = await account.get();
+      const userData: IUser = await getCurrentUser(user.$id);
+      updateUser(userData.id, userData.email, userData.league);
+      router.push(`/picks?leagueId=${userData.league[0]}`);
```

Comment:
This is assuming to use the first league? We want this to be dynamic.

Reply:
I agree.  However, there is not a page to list the leagues the user is a part of. AND then have them select the league.  Can't make this dynamic until we get that page which is why I have it picking the first one (for now).

Reply:
How can make this dynamic? The point of this ticket is to make it dynamic?

Reply:
Add a `// TODO:` comment to this so we know to come back to this.

Reply:
✅ Done!